src/test/java/com/google/api/codegen/testdata/ruby_gemspec_library.baseline

```diff
@@ -0,0 +1,26 @@
+============== file: library.gemspec ==============
+
+Gem::Specification.new do |s|
+  s.name          = 'library'
+  s.version       = '0.1.0'
+
+  s.authors       = ['Google, Inc.']
+  s.description   = 'Google Example Library API client for Ruby.'
+  s.email         = 'googleapis-packages@google.com'
+  s.files         = Dir.glob(File.join('lib', '**', '*.rb'))
+  s.homepage      = 'https://github.com/googleapis/googleapis'
+  s.license       = 'Apache-2.0'
+  s.platform      = Gem::Platform::RUBY
+  s.require_paths = ['lib']
+  s.required_ruby_version = '>= 2.0.0'
+  s.requirements << 'libgrpc ~> 1.0 needs to be installed'
+  s.summary       = 'GRPC library for service {{{api.name}}}'
+
+  s.add_dependency 'grpc', '~> 1.0'
+  s.add_dependency 'googleauth', '~> 0.5.1'
+  s.add_dependency 'google-common-protos', '~> 1.3.1'
+  s.add_dependency 'google-some-other-package-v1', '~> 0.2.1'
+
+  s.add_development_dependency 'bundler', '~> 1.9'
+  s.add_development_dependency 'rake', '~> 10.4'
+end
```

Comment:
The last 'no newline' style is not consistent among this file and `ruby_doc_gemspec_library.baseline`. Is it expected?